documentation/src/main/asciidoc/topics/proc_configuring_memory_offheap.adoc

```diff
@@ -5,29 +5,26 @@ heap space.
 
 .Procedure
 
-. Create a configuration for any type of cache.
-. Store cache entries in off-heap memory.
+. Modify your cache configuration to use off-heap memory.
 +
-* Declaratively: Add the `storage="OFF_HEAP"` attribute to the `memory` element.
-* Programmatically: Call the `storage(OFF_HEAP)` method in the `MemoryConfigurationBuilder` class.
+* Declarative: Add the `storage="OFF_HEAP"` attribute to the `memory` element.
+* Programmatic: Call the `storage(OFF_HEAP)` method in the `MemoryConfigurationBuilder` class.
 +
-. Configure eviction to limit the amount of off-heap memory that the cache can use. You can use eviction based on total number of entries (`max-count`) or maximum amount of memory (`max-size`).
-. Configure any binary encoding for cache entries. For optimal results, use the `application/x-protostream` MediaType to store entries in Protobuf format.
-. Create caches that use the configuration.
+. Configure `max-count` or `max-size` eviction to limit the amount of off-heap memory that the cache can use.
 
-.Declarative cache configuration
-
-[source,xml,options="nowrap"]
+.Declarative configuration
+[source,xml,options="nowrap",subs=attributes+,role="primary"]
 ----
 include::config_examples/max_size_offheap.xml[]
 ----
 
-.Programmatic cache configuration
-
-[source,java,options="nowrap"]
+.Programmatic configuration
+[source,java,options="nowrap",subs=attributes+,role="secondary"]
 ----
 include::code_examples/MaxCountOffHeap.java[]
```

Comment:
the `Programmatic` (uses max-count) does not match the `Declarative` (uses max-size) 
It would be nice to have the example separated
* prog + decl with max-count
* prog + decl with max-site

Also, it is not clear for me if the `<encoding media-type="application/x-protostream"/>` is required for off-heap. 
If not, just remove it from these examples

Reply:
thanks @pruivo good suggestions as always. I've made those changes.